docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,133 @@
+| `telemetrySubmitTopLevel` | String | No | Determines which nesting strategy to adopt for traces about top level traces and nested traces.<br><br>See the [Nested tracing strategies using Amazon XRay] section for more details about nesting strategies.<br><br>`ALWAYS_TOP_LEVEL` means that every JDBC call to the driver will generate a trace with no direct parent trace attached to it. If there is already an open trace being recorded by the application, no driver traces will be created.<br>`ALWAYS_NESTED` means that the wrapper will assume that there is an open trace in the application and the driver traces will all be attached to that open trace. If the user selects this value for the property and there is no open trace in the user application, no trace will be created.<br>`TOP_LEVEL` means that the application will look if there is an open trace in the users application prior to record telemetry data. If there is a current open trace, the traces created will be attached to that open trace. If not, all telemetry traces created will be top level.<br>`NO_TRACE` means that no traces will be created during the wrapper execution. | `TOP_LEVEL` |
+
+## Nested tracing strategies using Amazon XRay
+
+As you could see in the previous section, the AWS JDBC Driver allows an user to determine which strategy for nested traces to use when using Telemetry.
+
+Traces are hierarchical entities and it might be that the user application has already an open trace in a given sequence of code that connects to the AWS JDBC Driver. In this case, the Telemetry feature allows users to determine which strategy to use for the Telemetry traces generated when using the driver.
+
+A top level trace is a trace that has no link to any other parent trace, and is directly accessible from the list of submitted traces within XRay. When a trace is hierarchically linked to a parent trace, we say that this trace is nested. An example of nested traces are the individual plugin traces for a given JDBC call. All the individual plugin traces are linked to a parent trace for the JDBC call. Those nested traces are illustrated in picture 2.
+
+[Picture 1: Picture of a top level trace]
+
+[Picture 2: Picture of the plugin nested traces for a JDBC call]
+
+Applications that interact with the AWS JDBC Driver may or may not have already opened telemetry traces on their own. In this case, it is up to the user to determine how they want to mix both applications and driver traces.
+
+The `telemetrySubmitTopLevel` property of the AWS JDBC Driver allows users to determine that. Within that property, users can choose to submit the driver traces always as top level traces, to submit them nested to the application traces, or even to not submit the driver traces at all.
```

Comment:
```suggestion
Applications that interact with the AWS JDBC Driver may or may not have already opened telemetry traces on their own. In this case, it is up to the user to determine how they want to mix both application and driver traces.

This can be done using the AWS JDBC Driver's `telemetrySubmitTopLevel` property. This property allows users to choose to submit the driver traces always as top level traces, to submit them nested to the application traces, or even to not submit the driver traces at all.
```
?